dev/jobs/gen_control_ic.sh

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
+echo "gen_control_ic test passed."
+exit 0
```

Comment:
It would be worth adding the usual commands in here, even if it is a stub for now. See https://github.com/NOAA-EMC/global-workflow/blob/develop/dev/jobs/eupd.sh for an example of what this script should look like, minus changes related to the `job` variable and what JJOB script it would submit (which can be just a comment for now or you can add the anticipated JJOB script name now).